internal/dnfjson/dnfjson.go

```diff
@@ -115,6 +119,14 @@ func (s *Solver) Depsolve(pkgSets []rpmmd.PackageSet) ([]rpmmd.PackageSpec, erro
 	if err != nil {
 		return nil, err
 	}
+	// touch repos to now
+	now := time.Now().Local()
+	for _, r := range repoMap {
+		// ignore errors
+		_ = s.cache.touchRepo(r.Hash(), now)
+	}
+	s.cache.updateInfo()
```

Comment:
As suggested in previous patch, it would be nice if this was not needed. However, if we keep this, it would be nice if we could somehow log about the time spent so we would know in case it was a bottleneck.